bukuserver/api.py

```diff
@@ -101,16 +94,28 @@ def get(self, tag: T.Optional[str]):
         return {"name": tag, "usage_count": tags[1][tag]}
 
     def put(self, tag: str):
+        try:
+            form = forms.ApiTagForm.from_json(request.get_json())  # pylint: disable=no-member
+        except Exception as e:
+            current_app.log_exception(e)
+            return response.bad()
```

Comment:
I would rather return `response.bad(str(e))` but that means the `test_invalid_id` test will have to be parametrized differently to not check for the response message. Would you be open to only check for the status_code?
Btw are we ok with the fact that raised `expeptions.NotFound()` return no `status` in the response json?

Reply:
> to not check for the response message

Why shouldn't it check for the response message? I'm pretty sure you can use `response.bad()` (or a simplified equivalent thereof) to produce responses for test cases.

> Btw are we ok with the fact that raised expeptions.NotFound() (e.g. [line 100](https://github.com/jarun/buku/blob/db054e154dfe6cf26c47f92a8fa59504e39f1a48/bukuserver/api.py#L100) ) return no status in the response json?

I suppose the response format should stay consistent across the API…

Reply:
I  was concerned that if I returned to whole exception message it might change with time or the implementation... But OK, as commented above I will return simplified messages and test against them. I will use some sort of template (or simply constants, e.g. `TAG_NOT_VALID = "Tag is not valid"`) that can be reused to avoid duplicating strings between response.py and test_server.py

Reply:
You can use an [enum](https://docs.python.org/3.10/library/enum.html), I guess (e.g. `Message` or `Error`)